src/libs/SidebarUtils.ts

```diff
@@ -489,8 +478,8 @@ function getOptionData(
         result.alternateText = lastMessageText || formattedLogin;
     }
 
-    result.isIOUReportOwner = ReportUtils.isIOUOwnedByCurrentUser(result);
-    result.iouReportAmount = ReportUtils.getMoneyRequestReimbursableTotal(result);
+    result.isIOUReportOwner = ReportUtils.isIOUOwnedByCurrentUser(result as Report);
```

Comment:
@kubabutkiewicz Is it possible to get rid of assertion here?

Reply:
Its possible but I think its not worth it , `result` is of type `OptionItemData` which is basically Report + ReportAction type , so `result` will have all necessary fields for those function where I am using assertion. But changing those functions to accept type `OptionItemData` will require a lot of changes becuae they are using also other functions from ReportUtils